README.md

```diff
@@ -54,13 +55,23 @@ To build the library, you need to install:
 
 ### Build:
 KickCAT project is handled through CMake. To build the project, call CMake to configure it and then the build tool (default on Linux is make):
-  1. Configure the project (more information on https://cmake.org/cmake/help/latest/)
+  1. Create the build folder
+  ```
+  mkdir -P build
+  ```
+  2. Install conan and the dependencies with it (you may need to adapt the profile to suit your current installation). This step is mandatory for Windows, optional otherwise
+  ```
+  python3 -m venv kickcat_venv
+  ./kickcat_venv/bin/activate
+  pip install conan
+  conan install conan/conanfile_windows.txt -of=build/ -pr:h conan/profile_windows_x86_64.txt -pr:b conan/profile_linux_x86_64.txt --build=missing -s build_type=Release
```

Comment:
There is a mix of windows and linux profile in this command, is it normal ?

Reply:
Mmmm not really. It IS a working call but for cross compilation... I'll fix that before merge